### PR TITLE
Fixing ci and fix tests incorporating Tessa's changes, fix sourmash minhash import

### PR DESCRIPTION
**Pull request by pranathivemuri**
- Fixing ci github action miniconda import
- fix tests incorporating Tessa's changes -  https://github.com/czbiohub/orpheum/pull/101 closes issue #100 
- fix sourmash minhash import

Many thanks to contributing to czbiohub/orpheum!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Ensure the test suite passes with [pytest](https://docs.pytest.org/en/latest/) . (command to run: `pytest` or `make coverage` if you want to see which lines don't have tests yet)
 - [x] Make sure your code is linted and autoformatted using [black](https://github.com/psf/black) (`black . --check`).
 - [ ] Documentation in `usage.md` is updated
 - [ ] `README.md` is updated

### CODE REVIEW

File: tests/test_create_save_summary.py [hunk 1]
```diff
@@ -242,8 +244,13 @@ def test_generate_coding_summary(reads, data_folder, single_alphabet_ksize_true_
         "peptide_ksize": 7,
         "jaccard_threshold": 0.5,
     }
-
-    assert test_summary == true_summary
+    for key, value in test_summary.items():
+        if type(value) is str:
+            assert value == true_summary[key]
+        elif type(value) is float:
+            assert_almost_equal(value, true_summary[key], decimal=15)
```
Comment (bluegenes) [inline tests/test_create_save_summary.py:251]: ah - was this the actual issue with the 75% value above?

**Review by bluegenes — APPROVED** [review]
lgtm.

but are py 3.6 tests still failing?

File: tests/test_create_save_summary.py (hunk 1 above)
Comment (pranathivemuri) [reply tests/test_create_save_summary.py]: yes, it seems like 3.6 checks until a certain decimal point vs 3.7, 3.8

Comment (pranathivemuri) [reply tests/test_create_save_summary.py]: trying to fix that by only checking until a certain decimal point if a value is float

Comment (bluegenes) [reply tests/test_create_save_summary.py]: ah, makes sense!

**Pull request merged**